R/create_credentials.R

```diff
@@ -228,7 +279,22 @@ creds_internal <- function(user = NULL,
   )
 }
 
-#' Stops function is the system is not capable of using \pkg{keyring}
+#' Stops function if the **keyring** package isn't installed
+#'
+#' @noRd
+validate_keyring_available <- function(fn_name) {
```

Comment:
This function should also incorporate `validate_keyring_capable`

Reply:
Now done.